tietokanta/src/main/resources/db/migration/R__Suolapisteet.sql

```diff
@@ -0,0 +1,206 @@
+-- Funktiolle välitetään kaksi suolaustoteuman pistettä: alkupiste ja loppupiste, jossa suolankäyttö raportoidaan.
+-- Funktio muodostaa pisteiden perusteella tieosoitevälin ja jakaa sen osiin sen mukaan miten välille osuu suolankäytön rajoitusalueita ja rajoittamatonta aluetta.
+-- Rajoitusalueet ovat urakka- ja hoitovuosikohtaisia.
+-- Funktio palauttaa tiedot erilaisista alueista tieosoitevälillä ja kertoimet, joiden avulla loppupisteessä raportoitu suola voidaan jakaa eri alueiden kesken.
+CREATE OR REPLACE FUNCTION pistevalin_suolausalueet(piste1 POINT, piste2 POINT, urakka_id_ INTEGER, hk_alkuvuosi INTEGER)
```

Comment:
Tässä luodaan uusi signature, joten ajoon menee CREATE, eikä REPLACE. Tällöin käsittääkseni myös vanha signature jää kantaan eloon:
 pistevalin_suolausalueet(piste1 POINT, piste2 POINT, urakka_id_ INTEGER)
 Uhka vai mahdollisuus? :) Nähdäkseni vanha versio on erilainen, ja haluttaisiin tehdä DROP IF EXISTS?

Reply:
Mainio huomio! Lisäsin droppauksen v-migraatioon, niin ei jää loputtomiin pyörimään r-migraatiossa.